app/src/main/java/com/antosik/benchproject/app/popular/movies/view/PopularMoviesFragment.kt

```diff
@@ -24,6 +24,13 @@ class PopularMoviesFragment : Navigator<PopularMoviesFragmentNavigator>() {
         binding.lifecycleOwner = viewLifecycleOwner
         binding.viewModel = popularMoviesFragmentViewModel
         binding.recyclerAdapterForPopularMovies = PopularMoviesRecyclerAdapter(popularMoviesFragmentViewModel)
+        refreshListener(binding)
         return binding.root
     }
+
+    private fun refreshListener(binding: PopularMoviesFragmentBinding) {
```

Comment:
@BorysChajdas what do you think about this way to start refreshing the data from fragment using the method from ViewModel? At this moment this is the best that I came up with 😅.

Reply:
I think it could be set through XML Databinding

Reply:
I did it :D